webapp/src/js/bootstrapper/index.js

```diff
@@ -102,14 +102,19 @@
     });
   };
 
+  const createPurgingCheckpoint = () => {
+    return purger.info().then(purger.checkpoint);
+  };
+
   const initialReplication = function(localDb, remoteDb) {
     setUiStatus('LOAD_APP');
     const dbSyncStartTime = Date.now();
     const dbSyncStartData = getDataUsage();
 
-    return purger
-      .info()
-      .then(info => {
+    return createPurgingCheckpoint()
```

Comment:
What do you think if `createPurgingCheckpoint` is in the bootstrapper's [main function](https://github.com/medic/cht-core/blob/b2314e7f865a245a5ab5aa141d1903cc51fcf562/webapp/src/js/bootstrapper/index.js#L234) and have this function to only be about replication?

The main function is already big but still easier to see the step by step process: setting purging options, creating checkpoint (can these 2 be on the same function? ex: `initPurge`), checking if purge should happen, etc..

Reply:
We only want to create the checkpoint when doing initial replication, so I think it belongs in this function,